src/sync/installation.ts

```diff
@@ -249,7 +249,10 @@ const sendPayloadToJira = async (task: TaskType, jiraClient, jiraPayload, reposi
 			default:
 				await jiraClient.devinfo.repository.update(jiraPayload, {
 					preventTransitions: true,
-					operationType: "BACKFILL"
+					operationType: "BACKFILL",
+					auditLogsource: "BACKFILL",
+					entityAction: `${task.toUpperCase()}_BACKFILLED`,
```

Comment:
Should we append the BACKFILLED as we are already storing the source into audit log as "BACKFILL"?